exec/java-exec/src/main/java/org/apache/drill/exec/store/easy/json/JSONRecordReader.java

```diff
@@ -226,7 +226,8 @@ public int next() {
           }
           ++parseErrorCount;
           if (printSkippedMalformedJSONRecordLineNumber) {
-            logger.debug("Error parsing JSON in " + hadoopPath.getName() + " : line nos :" + (recordCount + parseErrorCount));
+            logger.debug("Error parsing JSON in {}: line nos: {}",
```

Comment:
What `nos` means?

Reply:
I suspect author means "numbers", but of course there is only one. Changed to just "line:".